lib/Drawers/Drawer.js

```diff
@@ -228,6 +229,18 @@ var MoonDrawer = module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	updateControlDrawersHeight: function(){
+		if(this.controlDrawerComponents !== null ) {
+			var value = this.calcDrawerHeight(),
+				node = this.$.controlDrawer.hasNode(),
+				cHeight = node.getBoundingClientRect().height + value;
+			this.applyStyle('height', cHeight + 'px');
```

Comment:
`applyStyle` is kind a heavy job for performance aspect.
Should we do those work in every animation step?